frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -86,8 +89,10 @@ class DatasetImportView extends React.PureComponent<Props, State> {
     messages: [],
     activeDataSourceEditMode: "simple",
     activeTabKey: "data",
+    savedDataSourceOnServer: null,
+    inferredDataSource: null,
     differenceBetweenDatasources: null,
-    previousDataSource: null,
+    didApplyDifferenceBetweenDatasources: false,
```

Comment:
Could you unify the spelling of dataSource in this file as best as possible, so that the S is uppercase?